scss/reprivata-theme/custom.scss

```diff
@@ -185,3 +195,56 @@ $display-font-family: $work-sans;
 $lead-font-size: $font-size-base * 1.125;
 $lead-font-weight: 400;
 $small-font-size: 0.889rem;
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
+
+// scss-docs-start input-btn-variables
+
+$input-btn-padding-y: 8px;
+$input-btn-padding-x: 16px;
+$input-btn-font-family: $open-sans;
```

Comment:
I must not have caught this in a design review, but using different fonts for different fonts for components vs display leads to a whole lot of issues like:
- Height and visual weight mismatch
- Incohesive fonts within a component. When do you use Work Sans vs Open Sans for a complex component like a select box?
- Incohesive fonts when embedding a component in a block of text or inside another component.

Mixing fonts for distinct pieces of information like headings, quotes, code, etc are generally fine. But its generally best for interactive components to inherit whatever the global default font is.